Newspack/NewspackTests/Folders/FolderManagerTests.swift

```diff
@@ -66,4 +66,19 @@ class FolderManagerTests: XCTestCase {
         success = folderManager.setCurrentFolder(url: FileManager.default.temporaryDirectory)
         XCTAssertFalse(success)
     }
+
+    func testListFolders() {
+        let path = "TestFolder"
+
+        var folders = folderManager.enumerateFolders(url: folderManager.currentFolder)
+        XCTAssertEqual(folders.count, 0)
+
+        _ = folderManager.createFolderAtPath(path: path, ifExistsAppendSuffix: true)
```

Comment:
By adding `@discardableResult` to `createFolderAtPath`'s signature you can get away without the `_ = xxxx` workaround

Reply:
True, but in normal usage the result should not be discardable. I don't mind the test looking a little ugly in this case. :) Thanks Jorge!